src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c

```diff
@@ -507,7 +507,7 @@ void func_809289E4(ObjTsubo* this, PlayState* play) {
                     s16 yawDiff = this->actor.yawTowardsPlayer - GET_PLAYER(play)->actor.world.rot.y;
                     s32 absYawDiff = ABS_ALT(yawDiff);
 
-                    if (absYawDiff > DEGF_TO_BINANG(120.0f)) {
+                    if (absYawDiff > CAM_DEG_TO_BINANG(120.0f)) {
```

Comment:
Hmm, does this work with the non camera specific `DEG_TO_BINANG` macro. If not, we should probably change it to use the hex literal until we figure that out.

Reply:
Yeah, I'd prefer just 0x10000 / 3 for now, as I think I mentioned last time.